dvc/stage/params.py

```diff
@@ -12,3 +12,4 @@ class StageParams:
     PARAM_METRICS = "metrics"
     PARAM_PLOTS = "plots"
     PARAM_DESC = "desc"
+    PARAM_JOBS = "jobs"
```

Comment:
Is this used anywhere?

Reply:
It used to be used for argument passing but deprecated now. We can remove it.